src/main/java/org/openpnp/machine/reference/vision/ReferenceBottomVision.java

```diff
@@ -114,6 +119,14 @@ public PartAlignmentOffset findOffsets(Part part, BoardLocation boardLocation, L
         return new PartAlignmentOffset(offsets,false);
     }
 
+    @Override
+    public boolean canHandle(Part part) {
```

Comment:
There may be a small issue here. canHandle is checking the part settings map directly, whereas findOffsets is calling getPartSettings(). getPartSettings creates a default part settings when called, whereas checking the map directly will fail if the part has not been previously configured. Is this what you intended, or would it be better to use getPartSettings()?

The main issue I see here is that this changes default behavior a bit. Previously, if the user has bottom vision enabled it will be enabled by default for all parts, and the user can disable it on a part by part basis, as needed. With your changes it will default to being disabled for all parts unless the user has previously configured the part settings for that part.

Thoughts?